feat(BUX-669): Removed transportType and changed README

Nazarii-4chain opened the pull request:
<!--
    Thank you for contributing to our project! Before you submit your Pull Request, please make sure you've completed the items in this checklist.
    Please check the boxes below by putting an x in the [ ] like so: [x]. You can do it right after creating the PR.
-->

# Pull Request Checklist

- [x] 📖 I created my PR using provided  : [CODE_STANDARDS](https://github.com/bitcoin-sv/spv-wallet-admin/blob/main/.github/CODE_STANDARDS.md)
- [x] 📖 I have read the short Code of Conduct: [CODE_OF_CONDUCT](https://github.com/bitcoin-sv/spv-wallet-admin/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] 🏠 I tested my changes locally.
- [ ] ✅ I have provided tests for my changes.
- [x] 📝 I have used conventional commits.
- [ ] 📗 I have updated any related documentation.
- [x] 💾 PR was issued based on the Github or Jira issue.

<!--
## PR Title as Conventional Commit

Your PR title should also be a [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/). This will assist us in our release process and in assigning semantic version numbers to releases.
-->

<!--
## Related Tickets & Documents

If your changes relate to or close an issue, please include them below. We like to follow [Github's guidance on linking issues to pull requests](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).

For instance, having the text "closes #1234" would link the current pull request to issue number 1234. And when we merge the pull request, Github will automatically close the issue.

- Related Issue #
- Closes #
-->

<!--
## Feature/Issue Details

Your pull request should be tightly connected to a feature or issue. Ensure that you've covered all necessary pathways and there's a possibility to test code behavior following the issue description or feature specification.
-->

Nazarii-4chain committed:
feat(BUX-669): Removed transpprtType and changed README

Nazarii-4chain committed:
Merge branch 'master' into BUX-669/UpdateAftreRenaiming

chris-4chain commented on src/App.test.js:
```diff
@@ -1,5 +1,5 @@
 import { render } from '@testing-library/react';
-import { App } from './App';
+import { App } from './App.jsx';
```
Is the `.jsx` part necessary?

Nazarii-4chain replied:
Removed

Nazarii-4chain committed:
feat(BUX-669): Removed .jsx

Nazarii-4chain committed:
Merge branch 'BUX-669/UpdateAftreRenaiming' of https://github.com/bitcoin-sv/spv-wallet-admin into BUX-669/UpdateAftreRenaiming

arkadiuszos4chain commented on src/pages/login.jsx:
```diff
@@ -114,13 +107,13 @@ const Login = () => {
+          {!!config.serverUrl
             ?
             (!config.hideServerUrl &&
               <>
```
Can we remove empty (`<>`) markups?

arkadiuszos4chain commented:
I don't understand the scope of the report that this PR is based on.

Nazarii-4chain committed:
feat(BUX-669): merge master

Nazarii-4chain committed:
feat(BUX-669): updated dependency

Nazarii-4chain commented:
> I don't understand the scope of the ticket that this PR is based on.

Edited Readme after renaming. Added proper links. Removed setTransport because we don't use it anymore. Updated dependnecy

Nazarii-4chain committed:
Merge branch 'main' into BUX-669/UpdateAftreRenaiming

arkadiuszos4chain commented on src/pages/login.jsx:
```diff
@@ -114,35 +107,20 @@ const Login = () => {
                 <TextField
                   fullWidth
-                  label={config.transportType}
+                  label={"http"}
```
remove '{' and '}'

Nazarii-4chain replied:
Removed

arkadiuszos4chain commented:
Why did you add empty `pages/admin-login.jsx`?

Nazarii-4chain committed:
feat(BUX-669): removed unused files

Nazarii-4chain committed:
Merge branch 'main' of https://github.com/bitcoin-sv/spv-wallet-admin into BUX-669/UpdateAftreRenaiming

Nazarii-4chain committed:
Merge branch 'BUX-669/UpdateAftreRenaiming' of https://github.com/bitcoin-sv/spv-wallet-admin into BUX-669/UpdateAftreRenaiming

Nazarii-4chain committed:
feat(BUX-669): removed ""

Nazarii-4chain commented:
> Why did you add empty `pages/admin-login.jsx`?

Removed

Nazarii-4chain committed:
feat(BUX-669): merge main

Nazarii-4chain committed:
Merge branch 'main' into BUX-669/UpdateAftreRenaiming
